site/public/js/forum.js

```diff
@@ -1420,3 +1426,95 @@ function updateSelectedThreadContent(selected_thread_first_post_id){
         }
     });
 }
+
+const FORUM_AUTOSAVE_KEY = `${window.location.pathname}-forum-autosave`;
+
+function autosaveKeyFor(textarea) {
+    return `${window.location.pathname}-${textarea.id}-forum-autosave`;
```

Comment:
So, for the general textarea... one shown in the bottom I think we should not be using `textarea.id` and instead, use something general word or something?

Reply:
Yup, you're right, I'll fix this